crates/router/src/connector/globalpay/response.rs

```diff
@@ -344,7 +344,7 @@ pub enum FingerprintPresenceIndicator {
 }
 
 /// Indicates where a transaction is in its lifecycle.
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
```

Comment:
You can even derive `Copy` for enums with no fields (all variants are unit variants), as copying them is cheap. However, if your enum has variants with fields which cannot be copied (say `String` fields), you'll have to switch over to cloning them.

Since most "status" enums have only unit variants, you can derive `Copy` on them without much thought.